package.json

```diff
@@ -12,8 +12,10 @@
   "author": "joey",
   "license": "ISC",
   "dependencies": {
+    "@types/cheerio": "^0.22.11",
```

Comment:
We want types to be a `devDependencies`

Reply:
This was automatically edited file.

Reply:
Uninstall the package utilizing `npm uninstall @types/cheerio` and then re-install it using the `--save-dev` flag. `npm install @types/cheerio --save-dev`